src/pages/home/assets/home.scss

```diff
@@ -1,57 +1,149 @@
-@import "~theme/base/variables";
+@import '~theme/base/variables';
+@import '~theme/base/utils';
+@import '~theme/base/mediaqueries';
 
-.homeContainer {
-  height: 100%;
-}
+$distance-height-variation-before: 20px;
+$distance-height-variation-after: 40px;
+
+// Sections with the text at the right
+@mixin isReverseSection() {
+    text-align: right;
+
+    header {
+        &:after {
+            right: 0;
+        }
+    }
 
-.mainHeaderBackground {
-  background: url('./main-header-background.png') no-repeat;
-  height: 100%;
-  width: 100%;
-  background-size: cover;
-  position: absolute;
-  z-index: -1; 
+    @include respond-to(pc) {
+        > div {
+            flex-direction: row-reverse;
+        }
+    }
 }
 
-.mainHeaderContent {
-  display: flex;
-  margin-left: 4em;
+.hero {
+    position: relative;
+    min-height: 50vh;
+    padding-top: 6rem;
```

Comment:
Please use `px` for margin & padding.

Reply:
This is in multiple instances, but let's be consistent :)

Reply:
This is a carry forward from #73 :)
I will fix this :)